src/modules/common/components/bars/topBar/hardwareWallet.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Tooltip from 'src/theme/Tooltip';
+import { useTranslation } from 'react-i18next';
+import Icon from 'src/theme/Icon';
+import { HW_CONNECTION_STATUS } from '@libs/hwServer/constants';
+import styles from './hardwareWallet.css';
+
+const Status = ({ status }) => (
+  <div className={`${styles.statusWrapper} ${styles[status]}`}>
+    <b>{status}</b>
+  </div>
+);
+
+const HardwareWallet = () => {
```

Comment:
Will we not receive the currentDevice from the TopBar? So we can hide this component if there is not currentHwDevice connected.

Reply:
I have added a comment for this too within the component. This way when the hook is consumed by the component, relevant information and be displayed and logic can be implemented @oskarleonard

Reply:
"component should return null" 
Isnt it nicer to follow the current logic we have in the Topbar, like <Network> and <AccountManagementDropdown>. 

Keeping the same conditions so one can easily see in the Topbar that is has a conditional show/hide